dhis-2/dhis-services/dhis-service-schema/src/main/java/org/hisp/dhis/schema/AbstractPropertyIntrospectorService.java

```diff
@@ -85,6 +95,18 @@
         .put( AnalyticalObject.class, BaseAnalyticalObject.class )
         .build();
 
+    private static final ImmutableSet<String> JSONB_TYPE_CLASSES = ImmutableSet.<String>builder()
+        .add( JsonBinaryType.class.getName() )
+        .add( JsonBinaryPlainStringType.class.getName() )
+        .add( JsonDeviceRenderTypeMap.class.getName() )
+        .add( JsonJobParametersType.class.getName() )
+        .add( JsonListBinaryType.class.getName() )
+        .add( JsonSetBinaryType.class.getName() )
+        .add( JsonEventDataValueSetBinaryType.class.getName() )
+        .add( JsonProgramRuleEvaluationEnvironmentSetBinaryType.class.getName() )
+        .add( JsonAttributeValueBinaryType.class.getName() )
+        .build();
+
     private final Map<Class<?>, Map<String, Property>> classMapCache = new HashMap<>();
```

Comment:
Wondering it we could instead check  if property's class extends `JsonBinaryType` 
People may forget to add new JsonB type into this list.

Reply:
I agree with you @vietnguyen . I removed my implementation from this PR, then we can re-implement with your suggestions later :)